### PR TITLE
Action API Readme Added

### PR DESCRIPTION
**Pull request by shreya024**

### CODE REVIEW

File: api-samples/alarms/alarms.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Title
```
Comment (patrickkettner) [inline api-samples/alarms/alarms.md:1]: was this intentionally added?

**Comment by patrickkettner** [comment]
Thanks so much for submitting the PR!
Would you be able to update it to the [readme template](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/README-template.md) thats been mandated for updates? It helps the project be more cohesive.

thanks again!

**Comment by shreya024** [comment]
> Would you be able to update it to the [readme template](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/README-template.md) thats been mandated for updates? It helps the project be more cohesive.

Hey @patrickkettner Thank you for reviewing. I have written the README as per the format of the readme template...is there any specific changes you are asking for?

**Comment by patrickkettner** [comment]
I apologize! I must have been looking at another PR. Thanks for the work!

**Review by patrickkettner — APPROVED** [review]

**Comment by shreya024** [comment]
> I apologize! I must have been looking at another PR. Thanks for the work!

Thank you for merging !

**Pull request merged**